CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing to Libero
+We encourage contributions of any kind.  By participating in this project, you agree to abide by the [code of conduct](https://libero.pub/code-of-conduct/).
+
+Contributions of code or documentation are covered by our [Contributor License Agreement (CLA)](https://libero.pub/governance/cla/). Please [sign the CLA](https://libero.pub/sign-cla) or ensure your employer has signed the CLA on your behalf.
```

Comment:
`s/ensure/make sure/`

Reply:
Might need to say what happens if it's not signed.

Reply:
THe CLA itself, and other examples just provide an email address for questions for anything other than the happy path of signing, so I've added that.